### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by lmntl27** [TobiasM95/WoW-Talent-Tree-Manager#17] [General]
Hey! I didn't know where to say this or the best way to send you this message. BUT THANK YOU SO MUCH. THIS PROGRAM IS AWESOME. 

I've always wanted something like this. This is perfect. If i had any coding skill i'd for sure help you out lol. But truly, you've done great work and wanted you to know its vastly appreciated

**Comment by Tyzam1** [comment]
I would also like to express my thanks for this super cool program. I was able to whip up a few fantasy class & spec trees in a single day with full description and pathing functionality. The tool is super easy.
I was blown away that you could drag the nodes around within the editor! Totally didn't expect that from this level of tool.
